test/integration/integration_kubernetes_test.go

```diff
@@ -416,3 +416,93 @@ kubernetes:
 	// https://github.com/kubernetes/kubernetes/pull/110479
 	asserter.droppedTargetCount(t, 1)
 }
+
+func Test_CuratedExperience(t *testing.T) {
+	t.Parallel()
+
+	k8sEnv := newK8sEnvironment(t)
+
+	// Create running pod
+	podBase := &corev1.Pod{
```

Comment:
nit: there is a func to create a standard pod IIRC might be it could be used to simplify this